util/tc.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+from __future__ import print_function
+from __future__ import absolute_import
```

Comment:
nit: combine these lines.
Import division here as well, so you don't need to explicitly type hint divisions below.

Reply:
Done